src/Raku/ast/statements.rakumod

```diff
@@ -1,7 +1,25 @@
 # Block or statement, used in statement prefixes which can take either a
-# block or a statement.
+# block or a statement (or an expression, which can always go in a
+# RakuAST::Statement::Expression)
 class RakuAST::Blorst
-  is RakuAST::Node { }
+  is RakuAST::Node {
+    method as-block() {
+        nqp::istype(self, RakuAST::Block)
```

Comment:
Instead of doing a type check here, RakuAST::Block could just override the method with an identity version.

Reply:
Good point!